### PR TITLE
NGSTACK-945 add aria label on search input

### PR DESCRIPTION
**Pull request by TTrakostanec**

### CODE REVIEW

**Review by zulicek — COMMENTED** [review]
možeš u sklopu ovog taska dodati role="status" na rezultate searcha, na <p class="search-text" ... />

**Comment by TTrakostanec** [comment]
@zulicek rijeseno u https://github.com/netgen/media-site/pull/217/commits/4913d1a510ef790033efc39c0b6d73aaa595ad9f

**Review by emodric — APPROVED** [review]

File: templates/themes/app/pages/search.html.twig [hunk 1]
```diff
@@ -8,14 +8,14 @@
             <div class="container">
                 <div class="search-inputs">
                     <div class="input-group">
-                        <input type="text" value="{{ search_text }}" name="searchText" id="Search" class="form-control" placeholder="{{ 'ngsite.search.placeholder'|trans }}" />
+                        <input type="text" aria-label="{{ 'ngsite.search.placeholder'|trans }}" value="{{ search_text }}" name="searchText" id="Search" class="form-control" placeholder="{{ 'ngsite.search.placeholder'|trans }}" />
                         <button type="submit" class="btn btn-sm btn-dark">{{ 'ngsite.search.button'|trans }}<i class="icon-search" aria-hidden="true"></i></button>
```
Comment (zulicek) [inline templates/themes/app/pages/search.html.twig:12]: bilo bi dobro i na ovaj button dodati aria-label, nešto tipa "Submit search"

Comment (TTrakostanec) [reply templates/themes/app/pages/search.html.twig]: done in https://github.com/netgen/media-site/pull/217/commits/63da769837d1bdd0a335ab637d03dd9874751cd5

**Pull request merged**